.github/codecov.yml

```diff
@@ -2,12 +2,10 @@ coverage:
   status:
     project:
       default:
-        target: auto
         # Only fail PR, if overall project codecov drops by 0.5%.
         threshold: 0.5%
```

Comment:
I think to be lenient we should make between 2-5%. If the patch stays at 80% I think that means we should never loose too much coverage, but still be quite lenient

Reply:
Also are you sure omitting `target: auto` is correct? I did a test on the madqn branch and got it working with these settings:

```
  status:
    project:
      default:
        target: auto
        threshold: 5%
    patch:
      default:
        target: 80%
```

However (and importantly) I put the file in the root dir, not in the .github folder

Reply:
> I think to be lenient we should make between 2-5%. If the patch stays at 80% I think that means we should never loose too much coverage, but still be quite lenient

Let's make it 1% for now and see if it causes issues.

Reply:
> Also are you sure omitting `target: auto` is correct? I did a test on the madqn branch and got it working with these settings:
> 
> ```
>   status:
>     project:
>       default:
>         target: auto
>         threshold: 5%
>     patch:
>       default:
>         target: 80%
> ```
> 
> However (and importantly) I put the file in the root dir, not in the .github folder

So when I validated the config (https://docs.codecov.com/docs/codecov-yaml#validate-your-repository-yaml), it said `target: auto` was a incorrect parameter. This was causing codecov to not read these settings in the past, now it does. 

Maybe try validate your madqn config?

Reply:
just validated and my config is valid, [here's the one on dqn](https://github.com/instadeepai/Mava/pull/843/files#diff-2b2bda0c32d4aa698c35a605120c7745c3433d4ad4210996bfc7f32e9fe9f078) and the codecov is passing even though project cov is down 0.89% 

```
   project:
      default:
        target: auto
        threshold: 1%
    patch:
      default:
        target: 80%
```